azure/lb.sh

```diff
@@ -120,6 +120,25 @@ fn azure_lb_addresspool_delete(name, group, lbname) {
 	)
 }
 
+fn azure_lb_addresspool_get_id(addrpoolname, resgroup, lbname) {
+	out, err <= (
+		az network lb address-pool show
+					--resource-group $resgroup
+					--lb-name $lbname
+					--name $addrpoolname
+					--output json
+					>[2=]
+	)
+
+	if $err != "0" {
```

Comment:
Yeah, I agree with @vitorarins that just ignoring the error returning empty string is bad.
But I know that for now is not straightforward to get stderr in a variable. I've improved this in https://github.com/NeowayLabs/nash/pull/227
On that PR, we can get stderr like this:

```sh
outstr, status <= cmd # stderr goes to /dev/stderr
outstr, errstr, status <= cmd # stderr now goes to $errstr variable
```

There's two ways to achieve the same semantics as above in current nash (but I dislike both) that we can use for now.

The first way (my preferable, but requires sane behaviour on the command being executed):

```sh
fn azure_lb_addresspool_get_id(addrpoolname, resgroup, lbname) {
        # redirects stderr into stdout
        # if the cli only print to /dev/stderr *in case of errors* than
        # everything's fine.
	out, status <= (
		az network lb address-pool show
					--resource-group $resgroup
					--lb-name $lbname
					--name $addrpoolname
					--output json
					>[2=1]
	)

	if $status != "0" {
                # there's no address pool for this lb
                # In this case $out contain the error 
		return "", $out
	}

	addresspoolid <= echo -n $out | jq -r ".id"
	return $addresspoolid, ""
}
```
The other way (the one that really sucks), is redirecting stderr to some file and reading it in case of error

```
errfile = "/tmp/az-blah-blah.err"
out, status <= az blah blah >[2] $errfile
if $status != "0" {
        errstr <= cat $errfile
        return "", $errstr
}

# use $out
```